backend/open_webui/utils/telemetry/metrics.py

```
@@ -0,0 +1,110 @@
+        View(
+            instrument_name="http.server.requests",
+            attribute_keys=["http.method", "http.route", "http.status_code"],
+        ),
+    ]
+
+    provider = MeterProvider(
+        resource=Resource.create({SERVICE_NAME: OTEL_SERVICE_NAME}),
+        metric_readers=list(readers),
+        views=views,
+    )
+    return provider
```

Ruff (0.11.9)

Unnecessary assignment to `provider` before `return` statement

Remove unnecessary assignment

(RET504)